modules/contact.js

```diff
@@ -39,6 +39,7 @@
 var EXPORTED_SYMBOLS = ["ContactManager", "Contacts", "defaultPhotoURI"];
 
 ChromeUtils.import("resource:///modules/StringBundle.js"); // for StringBundle
+ChromeUtils.import("resource:///modules/DisplayNameUtils.jsm"); // for DisplayNameUtils.getCardForEmail
```

Comment:
nit: Please use the alternate format here, as it is better for ESLint to detect the globals:
```suggestion
const {DisplayNameUtils} = ChromeUtils.import("resource:///modules/DisplayNameUtils.jsm");
```